### DISCUSSION TITLE
Module strust

### DISCUSSION THREAD

**Discussion by rainerleber** [sap-linuxlab/community.sap_libs#76] [Ideas]
### Summary

A module which manage certificates with strust

### Issue Type

Feature Idea

### Component Name

strust, pyrfc

### Additional Information

-

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

**Comment by bbloemer-cg** [comment]
Hi,

we are actually looking for something like this at the moment to ease up certificate management in our landscape. So I would really love to use this. 
As @jfilak mentioned in [community.sap ](https://github.com/ansible-collections/community.sap/issues/19) maybe he can help on this one?

best regards
Benedikt
